ny-portal/src/fields/code-example/CodeExampleField.ts

```diff
@@ -0,0 +1,15 @@
+import { globSync } from "node:fs";
+import { readFile } from "node:fs/promises";
+import path, { resolve } from "node:path";
+import type { Field } from "payload";
+import { CodeExampleInput } from "./CodeExampleInput";
```

Comment:
Her er det vel bare `Field` som brukes nå?

```suggestion
import type { Field } from "payload";
```

Reply:
Ja, ser ikke ut som linteren min kjører for denne mappa :)